LICENSE

```diff
@@ -210,170 +210,176 @@
+    MathJax (3rdparty/mkldnn/doc/assets/mathjax)
           Copyright (c) 2009-2018 The MathJax Consortium
           Copyright (c) 2015-2017 Martin Hensel
           Copyright (c) 2007, Apostolos Syropoulos (<asyropoulos@yahoo.com>)
-    28. material-design-icons - For details, see ./docs/python_docs/themes/mx-theme/mxtheme/static/material-design-icons-3.0.1/iconfont/
-    29. Roboto fonts - For details, see ./docs/python_docs/themes/mx-theme/mxtheme/static/font/Roboto/
+    material-design-icons - For details, see ./docs/python_docs/themes/mx-theme/mxtheme/static/material-design-icons-3.0.1/iconfont/
+    Roboto fonts - For details, see ./docs/python_docs/themes/mx-theme/mxtheme/static/font/Roboto/
+    bfloat16 - For details, see 3rdparty/tvm/3rdparty/bfloat16/bfloat16.cc
+          Copyright (c) 2019 by Contributors
 
     =======================================================================================
-    MIT licenses
+    MIT license
     =======================================================================================
 
-    1. Fast R-CNN  - For details, see example/rcnn/LICENSE
+    Fast R-CNN  - For details, see example/rcnn/LICENSE
          Copyright (c) Microsoft Corporation
-    2. Faster R-CNN - For details, see example/rcnn/LICENSE
+    Faster R-CNN - For details, see example/rcnn/LICENSE
          Copyright (c) 2015 Microsoft Corporation
-    3. tree_lstm - For details, see example/gluon/tree_lstm/LICENSE
+    tree_lstm - For details, see example/gluon/tree_lstm/LICENSE
          Copyright (c) 2017 Riddhiman Dasgupta, Sheng Zha
-    4. OpenMP - For details, see 3rdparty/openmp/LICENSE.txt
-         Copyright (c) 1997-2016 Intel Corporation
-    6. HalideIR - For details, see 3rdparty/tvm/3rdparty/HalideIR/LICENSE
-         Copyright (c) 2016 HalideIR contributors
-         Copyright (c) 2012-2014 MIT CSAIL, Google Inc., and other contributors
-         Copyright (c) 2016-2018 by Contributors
-    7. ONNX-TensorRT - For details, see 3rdparty/onnx-tensorrt/LICENSE
+    cma.h, cma_api_impl.h - For details, see 3rdparty/tvm/3rdparty/cma/cma.h
+         COPYRIGHT (C) 2017 Institute of Electronics and Computer Science (EDI), Latvia.
+    ONNX-TensorRT - For details, see 3rdparty/onnx-tensorrt/LICENSE
          Copyright (c) 2018, NVIDIA CORPORATION. All rights reserved.
          Copyright (c) 2018 Open Neural Network Exchange
-    8. ONNX-TensorRT - For details, see 3rdparty/onnx-tensorrt/third_party/onnx/LICENSE
+    ONNX-TensorRT - For details, see 3rdparty/onnx-tensorrt/third_party/onnx/LICENSE
          Copyright (c) Facebook, Inc. and Microsoft Corporation.
-    9. clipboard.js - Refer to https://zenorocha.github.io/clipboard.js
+    clipboard.js - Refer to https://zenorocha.github.io/clipboard.js
          Licensed MIT © Zeno Rocha
-    10. clipboard.min.js - Refer to https://zenorocha.github.io/clipboard.js
+    clipboard.min.js - Refer to https://zenorocha.github.io/clipboard.js
```

Comment:
> In LICENSE, add a pointer to the dependency's license **within the distribution** and a short note summarizing its licensing

https://infra.apache.org/licensing-howto.html

The pointer is not included in the distribution

Reply:
what is needed? a path to the 3rdparty/... directory?

Reply:
One needs to put the license somewhere in the distribution and point to it

Reply:
The website can go down, that's why Apache wants it to be in the distribution

Reply:
It's the standard MIT license (included in Appendix), and the copyright line is included in LICENSE. Isn't this enough?

Reply:
Updated to reference source file, which contains copyright and license notice.